### DISCUSSION TITLE
[1.7.3] 2022-08-18

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#363] [General]
### Global
[MAIN] Fix stack overflow when formating SPIFFS on first boot after initial flash (Fixes #359)

# Download are now on the new itch.io page: https://darthcloud.itch.io/blueretro

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.7.3'>[1.7.3] 2022-08-18</a>.</em>

**Comment by rafaguel** [comment]
Hi, Darthcloud.

First of all, thank you for the job and dedication.

I just buy a core and two cables for a renomad seller in alliexpress, one cable for psx and work well in ps1 and ps2. The other one is for GameCube, that one dont work in the GC from the Wii ports. The seller said it is because the cable will only function in originally (and purple) GC, any other system (like wii) dont will. But i dont that before the shopping, anyway i saw a lot of videos show and teacher do a GC adapter and use in wii ports via cable, other ones do internal.

I just start to research and saw here that user EduardoYagami have the same or almost equal problem. He said that work well in 1.2.1 FW and you said for he use the 1.6.3 FW and work very well.

I try 1.2.1, 1.6.3, 1.7.1 and 1.7.3 and no one work. The seller dont know what do and follow stating that i bought a GC cable and not a wii, and work only in a native old gamecube console.

What I can do in that case? I try four bin in all FW version: gb spiffs in 1.21 (the only one), in 1.6.3, 1.7.1 and 1.7.3 i try gc external, wii external, wii internal and universall. No sucess.

I have other question, about the N64 memory pack, i do a internal mod for all controllers (i do my self) and before saw that possibility, i update the Fw and try it, but always have the warning of problem in memory pak, not all game repair, and whan this happing, i cant use the memory pak, i try mk4 and work 3 times well and 1 the error, SF Rush 2 times getting error, i creat a perfil and cant save after the runnings, said that i remove the memory pak, when back to initial screen, i can repair. at the last Mystical Ninja Goemon 4 times and only 1 i can use the pak, other ones said that i dont have inserted memory pak.

Thanks again, and sory the poor english.

**Reply by gekkoton** [reply]
> 
> 
> I just buy a core and two cables for a renomad seller in alliexpress, one cable for psx and work well in ps1 and ps2. The other one is for GameCube, that one dont work in the GC from the Wii ports. The seller said it is because the cable will only function in originally (and purple) GC, any other system (like wii) dont will. But i dont that before the shopping, anyway i saw a lot of videos show and teacher do a GC adapter and use in wii ports via cable, other ones do internal.
> 
> I just start to research and saw here that user EduardoYagami have the same or almost equal problem. He said that work well in 1.2.1 FW and you said for he use the 1.6.3 FW and work very well.
> 
> I try 1.2.1, 1.6.3, 1.7.1 and 1.7.3 and no one work. The seller dont know what do and follow stating that i bought a GC cable and not a wii, and work only in a native old gamecube console.
> 
> What I can do in that case? I try four bin in all FW version: gb spiffs in 1.21 (the only one), in 1.6.3, 1.7.1 and 1.7.3 i try gc external, wii external, wii internal and universall. No sucess.
> 

I also bought a Core and GC cables from Aliexpress and have been working great on my Wii. I'm using 1.7.3 firmware. I think your Core or cable may be the issue.

**Reply by rafaguel** [reply]
Thanks, Gekkoton. I think it too, and how the core work with psx cable (i have only psx and that GC) i hope that the cable constuction.

I will wait of Darth Cloud telling for us if can the have problem to limit some (cable) especific systems.

I try to reprograming the core, but i have a issue on load the boot, partition table, and FW process, the program don enter in SYNC, stay in IDLE. So I stop and talk with seller, they tell me to return the cable because he is only fo the old one purple GC.

Wherever, i also think it is a cable problem, hard to be a core since he work with psx.

Thank you again by your report.

Edit.: The core turn on normally, with GC cable connected on wii.

**Comment by rafaguel** [comment]
Hi again.

I look in jack of adaptor cable and saw only 4 metallic pin, so i open the pinout part and saw they have only 4 wire for the jack. By the tutorial in that page, Its missing one wire.

I dont know If is right ir wrong.
![IMG_20221105_061530](https://user-images.githubusercontent.com/117371307/200128127-d1c9c8d5-f4a8-43d5-84d3-77fcaa16e937.jpg)
![IMG_20221105_061019](https://user-images.githubusercontent.com/117371307/200128142-0d8b4324-642d-46db-948a-c662b39fd6d6.jpg)
![IMG_20221105_060804](https://user-images.githubusercontent.com/117371307/200128149-68d4fe17-96b0-4a48-b0cb-77eba9a1e373.jpg)

**Comment by rafaguel** [comment]
For N64 memory pak, just change memory/rumble pak and bank (if necessary) looking the buttons of your controller im mapping. Sometimes the rumble is set other ones the memory, i dont know why chance, but one solution is enter in memory pak manager before of start the game and see if the controller pak is set, if dont, have game that cant let you chance and force you to restart the game.

**Comment by rafaguel** [comment]
Hi friends, before a lot of work, i do the adaptar work.

The tip stay in a Brazilian forum colled outer space.

Thank you.
